docs-src/index.md

```diff
@@ -1,36 +1,52 @@
 # Introduction
 
-Service APIs is the evolution of Kubernetes APIs that relate to `Services`,
-e.g. `Ingress`. The project is part of Kubernetes, working under
-[SIG-NETWORK][sig-network]. Use the following resources to learn more about
-Service APIs.
+Service APIs is an open source project, working under
+[SIG-NETWORK][sig-network], that is evolving service networking APIs within the Kubernetes ecosystem. Service APIs are the APIs you use to expose your Kubernetes applications - Services, Ingress, and more. The Gateway API is the primary API being developed to unify service networking under a core set of networking APIs.
+
+### What is the Gateway API?
+
+The Gateway API is an effort by a consortioum of companies and users to evolve the current Kubernetes Ingress API. Gateway aims to improve upon Ingress by becoming a more expressive, extensible, role-oriented API while remaining a generic routing API that has many implementations and broad industry support. 
```

Comment:
All of our existing text in markdown files is approximately 80 column long. Could you please wrap lines in this patch accordingly?

Reply:
Good catch - if you happen to be using VSCode I highly recommend [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) to do this automatically.

Reply:
rewrapped to 80 char